source/stylesheets/scss/furatto/_text_alignment.scss

```diff
@@ -0,0 +1,32 @@
+// Text Alignment
+
+.text {
+
+  &.left {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets